src/components/backup/BackupConfirmPasswordStep.js

```diff
@@ -133,14 +133,10 @@ export default function BackupConfirmPasswordStep() {
     []
   );
 
-  const onError = useCallback(
-    msg => {
-      passwordRef.current?.focus();
-      setIsWalletLoading(null);
-      DelayedAlert({ title: msg }, 500);
-    },
-    [setIsWalletLoading]
-  );
+  const onError = useCallback(msg => {
+    passwordRef.current?.focus();
+    DelayedAlert({ title: msg }, 500);
+  }, []);
```

Comment:
Can we add explicit type or explicit any here? React 18 doesn't like that and the bump PR is just around the corner